### PR TITLE
WEBDEV-7036 Integrate smart results into collection browser

### PR DESCRIPTION
**Pull request by rebecca-shoptaw**
Adds slot for smart results to collection browser, along with styling ("All results" + extra spacing), that displays conditionally if `showSmartResults` is set to true.

### CODE REVIEW

File: src/collection-browser.ts [hunk 1]
```diff
@@ -652,18 +655,33 @@ export class CollectionBrowser
...
-        ${this.displayMode === `list-compact` && this.totalResults
-          ? this.listHeaderTemplate
+      <div
+        id="right-column"
+        class="column ${this.showSmartResults ? 'smart-results-spacing' : ''}"
+      >
+        ${this.showSmartResults
+          ? html`<div id="smart-results">
+              <slot name="smart-results"></slot>
+            </div>`
```
Comment (latonv) [inline src/collection-browser.ts:665]: Q: Is the wrapper div necessary here? Could we instead just apply the extra margin to the results section below?

File: package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "description": "The Internet Archive Collection Browser.",
   "license": "AGPL-3.0-only",
   "author": "Internet Archive",
-  "version": "2.7.0",
+  "version": "2.7.1-alpha.9",
```
Comment (latonv) [inline package.json:6]: Just a reminder not to merge in the alpha version!

**Review by latonv — APPROVED** [review]
LGTM, no blockers!

File: package.json (hunk 2 above)
Comment (rebecca-shoptaw) [reply package.json:6]: Thank you!!

**Pull request merged**